Projects/Contracts/Sources/Journeys/ContractInformation.swift

```diff
@@ -318,7 +318,16 @@ struct ContractInformationView: View {
                     hSection {
                         hButton.LargeButton(type: .ghost) {
                             terminationContractStore.send(
-                                .startTermination(contractId: id, contractName: contract?.exposureDisplayName ?? "")
+                                .startTermination(
+                                    contractId: id,
```

Comment:
Can we add this 2 properties to the config? Actually one since
`contract?.exposureDisplayName ?? "" `
is redundant.